Automattic/wp-calypso
# Agency Dashboard: Make the Add button selectable

#### Proposed Changes

This PR transforms the `Add` button in the dashboard into a selectable option. This will allow users to select multiple licenses for a single site.

**Note:** This change should be available only for our development environment. No regression should be caused on other environments.

What's included in this PR:
- Update the `Add` link to be a select option
- Allow multiple `Add` options on the same site row to be selected
- Disable the `Add` options on other site rows when a site add option is selected

**Screenshot**

![image](https://user-images.githubusercontent.com/1749918/198999869-f44c68e1-280d-45a0-97c4-dd4127835bbe.png)

#### Testing Instructions
- Checkout this branch locally
- Run yarn `start-jetpack-cloud`
- Create a new JN site
- Connect Jetpack
- Visit http://jetpack.cloud.localhost:3000/dashboard
- Verify that you can select backup or scan license
- Verify that you can deselect any of your already selected licenses
- Verify that the other rows (sites) are disabled when you have a license selected for a given site. If you click on any disabled row, the selected licenses should be reset.
- Verify that the UI/UX works well on the mobile version as well
- Visit the Jetpack Cloud live link (horizon environment) in this PR and verify that `Add` still works as before for backup and scan (no regression)

#### Pre-merge Checklist






Related to 1203126240279377-as-1203126240279381
